src/components/network/network-map-tab.tsx

```diff
@@ -811,9 +812,21 @@ export const NetworkMapTab = ({
             // load root node geodata
             loadRootNodeGeoData();
         }
-        if (isRootNodeGeoDataLoaded && isMapEquipmentsInitialized) {
-            updateMapEquipmentsAndGeoData();
+        // manual reload
+        if (refIsMapManualRefreshEnabled.current && isInitialized) {
+            // this reloads the mapEquipments when, in manual refresh mode, the current node is built.
+            if (
+                isSameNode(previousCurrentNode, currentNode) && // must be the same node
+                !previousNodeStatus && // must pass from unbuilt ...
+                isCurrentNodeBuiltRef.current // to built
+            ) {
+                updateMapEquipmentsAndGeoData();
+            }
+            // return to avoid update in manual reload or to avoid double upate
+            return;
         }
```

Comment:
```suggestion
        if (!(refIsMapManualRefreshEnabled.current && isInitialized
                isSameNode(previousCurrentNode, currentNode) && // must be the same node
                !previousNodeStatus && // must pass from unbuilt ...
                isCurrentNodeBuiltRef.current // to built)
        ) {
            return;
        }
```

I would make something like this to keep the same structure as the rest of the useEffect
The condition I've suggested might be wrong, it's mostly to illustrate my point
Of course, keep the comments as you did before

Reply:
I prefer the previous version to manage clearly the manual mode to avoid `updateMapEquipmentsAndGeoData` and it's exception inside when node is built.